pcsx2-qt/Themes.cpp

```diff
@@ -55,6 +55,16 @@ void QtHost::SetStyleFromSettings()
 		qApp->setStyle(QStyleFactory::create("Fusion"));
 		qApp->setStyleSheet(QString());
 	}
+#ifdef _WIN32
+	else if (theme == "windowsvista")
+	{
+		// setPalette() shouldn't be necessary, as the documentation claims that setStyle() resets the palette, but it
+		// is here, to work around a bug in 6.4.x and 6.5.x where the palette doesn't restore after changing themes.
+		qApp->setPalette(QPalette());
```

Comment:
Can you please move this between lines 49 and 50? It's repeated on several of the branches here.

Reply:
Done.